cryptostore/data/parquet.py

```diff
@@ -21,7 +21,7 @@ class Parquet(Store):
 
     default_path = lambda self, exchange, data_type, pair: f'{exchange}/{data_type}/{pair}'
 
-    def __init__(self, exchanges, config=None):
+    def __init__(self, exchanges, buffer, config=None):
```

Comment:
this shouldnt be a required arg, it should be optional, or in kwargs

Reply:
@bmoscon : done